src/DssSpell.t.sol

```diff
@@ -859,4 +888,77 @@ contract DssSpellTest is DssSpellTestBase {
     }
 
     // SPELL-SPECIFIC TESTS GO BELOW
+
+    function testDirectSparkMorphoIntegration() public {
+        bytes32 ilk         = "DIRECT-SPARK-MORPHO-DAI";
+        address vault       = 0x73e65DBD630f90604062f6E02fAb9138e713edD9;
+        address operator    = 0x298b375f24CeDb45e936D7e21d6Eb05e344adFb5;
+        uint256 debtCeiling = 100 * MILLION * WAD;
+
+        _vote(address(spell));
+        _scheduleWaitAndCast(address(spell));
+        assertTrue(spell.done());
+
+        D3MHubLike hub = D3MHubLike(addr.addr("DIRECT_HUB"));
+        D3MMomLike mom = D3MMomLike(addr.addr("DIRECT_MOM"));
+        D3M4626PoolLike pool = D3M4626PoolLike(addr.addr("DIRECT_SPARK_MORPHO_DAI_POOL"));
+        D3MOperatorPlanLike plan = D3MOperatorPlanLike(addr.addr("DIRECT_SPARK_MORPHO_DAI_PLAN"));
+        D3MOracleLike oracle = D3MOracleLike(addr.addr("DIRECT_SPARK_MORPHO_DAI_ORACLE"));
+
+        { // Do a bunch of sanity checks of the values that were set in the spell
+            (address _pool, address _plan, uint256 tau, uint256 _culled,) = hub.ilks(ilk);
+            assertEq(_pool, address(pool));
+            assertEq(_plan, address(plan));
+            assertEq(tau, 7 days);
+            assertEq(_culled, 0);
+            assertEq(hub.vow(), address(vow));
+            assertEq(hub.end(), address(end));
+            assertEq(mom.authority(), address(chief));
+            assertEq(pool.vault(), vault);
+            assertEq(pool.redeemable(), vault);
+            assertEq(plan.operator(), operator);
+            assertEq(plan.wards(address(mom)), 1);
+            assertEq(plan.active(), true);
+            assertEq(oracle.hub(), address(hub));
+            (address pip,) = spotter.ilks(ilk);
+            assertEq(pip, address(oracle));
+            assertEq(vat.wards(address(hub)), 1);
+        }
+
```

Comment:
This should have been mentioned, but you need to set the supply queue before transferring funds. Here is it being done in the integration test: https://github.com/makerdao/dss-direct-deposit/blob/multisig-plan/src/tests/integration/MetaMorpho.t.sol#L129

Here is what it should look here roughly:

```
// Add top top
interface IMetaMorpho {
    function setSupplyQueue(bytes32[] calldata newSupplyQueue) external;
}

// Add to this section
bytes32[] memory newSupplyQueue = new bytes32[](1);
newSupplyQueue[0] = 0x1247f1c237eceae0602eab1470a5061a6dd8f734ba88c7cdc5d6109fb0026b28;
vm.prank(0x298b375f24CeDb45e936D7e21d6Eb05e344adFb5);
IMetaMorpho(vault).setSupplyQueue(newSupplyQueue);
```